.github/workflows/documentation.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
       - uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
         with:
           persist-credentials: false
-      - uses: docker://cilium/docs-builder:latest
+      - uses: docker://qmonnet/docs-builder:latest
```

Comment:
Yes and yes. I don't have permission to push so I'll need your help once reviews are good and we consider the PR in good shape. Once pushed, I can update this commit to point to the specific image (since we merged https://github.com/cilium/cilium/pull/19356).

Reply:
Now updated to point to the image you pushed, thanks!